examples/order-polytope-basics/README.md

```diff
@@ -0,0 +1,18 @@
+## Compilation
+In folder examples, first run cmake, to create the makefile:
+
+```bash
+cmake .
+```
+
+Then, in folder examples/order-polytope-basics compile and build using the makefile:
+
+```bash
+make
+```
+
+## Usage:
+```bash
+ ./order_polytope poset_data.txt
+```  
+where `poset_data.txt` is the file containing the poset data.
```

Comment:
could you comment on how this file represents a poset?

Reply:
Added instructions regarding the representation of poset data in the file.